modules/S115_usermode_emulator.sh

```diff
@@ -127,7 +127,7 @@ S115_usermode_emulator() {
           continue
         fi
 
-        if [[ "${BIN_BLACKLIST[*]}" == *"$(basename "${FULL_BIN_PATH}")"* ]]; then
+        if echo "${BIN_BLACKLIST[@]}" | grep -q -F -w "$(basename "${FULL_BIN_PATH}")"; then
```

Comment:
I think we do not need the extra echo and grep here:
```
if [[ "${BIN_BLACKLIST[*]}" == *" $(basename "${FULL_BIN_PATH}") "* ]]; then
```

Does this fit your solution?

Reply:
I don't know if it does. Does it work with the first and last item as well? 

I don't have time for any more testing. I just know that my proposed solution works. Feel free to implement whichever you like that works.